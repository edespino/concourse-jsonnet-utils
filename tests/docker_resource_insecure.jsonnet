local concourse = import 'concourse.libsonnet';

{
  docker: concourse.DockerResource('dev-container', 'registry:5000/dev-container', allow_insecure = true),
}
