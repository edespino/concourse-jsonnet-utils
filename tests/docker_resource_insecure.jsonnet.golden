{
   "docker": {
      "name": "dev-container",
      "source": {
         "insecure_registries": [
            "registry:5000"
         ],
         "repository": "registry:5000/dev-container",
         "tag": "latest"
      },
      "type": "docker-image"
   }
}
